package.json

```diff
@@ -36,6 +36,7 @@
     "prettier:write": "prettier --list-different \"**/*.{js,ts,tsx,scss,md,mdx}\" --write",
     "start": "yarn themes:generate && yarn dev --watch",
     "start:noTsCheck": "yarn start --env noTsCheck=1",
+    "start:noLint": "yarn start --env noTsCheck=1 --env noLint=1",
```

Comment:
can we just merge these two? happy to deprecate/remove noTsCheck in favor of noLint, or just alias them.

Reply:
Shall we name the new one `noCheck` and remove `noTsCheck`?

Reply:
If we're making a 'breaking change' to the commands (which I think is fine), I'm happy for `noLint` to _replace_ `noTsCheck`. `noLint` is an easier to understand name imho